svg-tag-mode.el

```diff
@@ -64,39 +64,39 @@
 (eval-when-compile (require 'subr-x))
 
 (defvar svg-tags nil)
-(defvar active-svg-tags nil)
+(defvar svg-tags--active-tags nil)
```

Comment:
I guess `tags` at the end is not necessary now.

Reply:
It really isn't but since it's, actually a cached copy of `svg-tags` (the name doesn't follow the naming conventions btw, should be smth like `svg-tag-tags` or smth)